app/register/page.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import RegisterForm from "@/components/sections/register-form";
+import RegisterContent from "@/components/sections/register-content";
+
+const RegisterPage = () => {
+  return (
+    <div className="container flex flex-col md:flex-row justify-center md:justify-between md:items-center h-screen">
+      <RegisterContent />
```

Comment:
RegisterContent bilan LoginContent bir xil faqat bitta text da farq qiladi shunga AuthContent degan file ochib ichiga props berib yuborish kerak ya'ni `isLogin` bu `boolean` qabul qiladi agar `true` bo'lsa `Sign in` else `Sign up` bo'ladi. Shunda kod qaytarilishining oldi ancha olingan bo'ladi